src/js/shot-generator/Character.js

```diff
@@ -72,37 +74,62 @@ const characterFactory = data => {
   let mesh
   let skeleton
   let armatures
-
+  let parentRotation = new THREE.Quaternion()
+  let parentPosition = new THREE.Quaternion()
   mesh = data.scene.children.find(child => child instanceof THREE.SkinnedMesh) ||
          data.scene.children[0].children.find(child => child instanceof THREE.SkinnedMesh)
 
   armatures = data.scene.children[0].children.filter(child => child instanceof THREE.Bone)
-
+  if (armatures.length === 0 ) {  // facebook export is different
+    //console.log('searching in: ',data.scene.children[0].children[0].children)
+    armatures = data.scene.children[0].children[0].children.filter(child => child instanceof THREE.Bone)
+    for (var bone of armatures)
+    {
+      //console.log('this scale: ', data.scene.children[0].children[0])
+      //bone.scale.multiply(data.scene.children[0].children[0].scale)
+      bone.scale.set(1,1,1)
+      bone.quaternion.multiply(data.scene.children[0].children[0].quaternion)
+      //bone.position.multiply(data.scene.children[0].children[0].scale)
+      bone.position.set(bone.position.x,bone.position.z,bone.position.y)
+      parentPosition = bone.position.clone()
```

Comment:
Hmm. I don't quite understand this. `parentPosition` is set every time through the loop? So it's essentially just being set to a clone of the last bone in the `armatures` array?

Reply:
well yes, but our armatures array is just the root bone(we're not going through the hierarchy), and that's the position we need

Reply:
changed to armatures[0].position.clone()